xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -931,8 +931,10 @@ CPVREpgInfoTagPtr CPVRTimerInfoTag::GetEpgInfoTag(bool bCreate /* = true */) con
           }
         }
 
-        if (!m_epgTag)
+        if (!m_epgTag && m_epTagRefetchTimeout.IsTimePast())
         {
+          m_epTagRefetchTimeout.Set(30000); // try to fetch missing epg tag from backend at most every 30 secs
```

Comment:
I'd probably use a constant

Reply:
Will do if the value gets used at more than one place.